Fix error: implicitly-declared 'constexpr ...' is deprecated [-Werror=deprecated-copy]
---
--- a/inference-engine/thirdparty/clDNN/src/gpu/memory_gpu.cpp
+++ b/inference-engine/thirdparty/clDNN/src/gpu/memory_gpu.cpp
@@ -127,8 +127,7 @@
             CLDNN_ERROR_MESSAGE("2D image allocation", "unsupported image type!");
     }

-    cl::ImageFormat imageFormat(order, type);
-    _buffer = cl::Image2D(_context->context(), CL_MEM_READ_WRITE, imageFormat, _width, _height, 0);
+    _buffer = cl::Image2D(_context->context(), CL_MEM_READ_WRITE, cl::ImageFormat(order, type), _width, _height, 0);

     if (reset) zero_image();
 }
